florestad/src/json_rpc/server.rs

```diff
@@ -116,7 +116,8 @@ impl<Blockchain: RpcChain> RpcImpl<Blockchain> {
     }
 
     fn load_descriptor(&self, descriptor: String) -> Result<bool> {
-        let Ok(mut parsed) = parse_descriptors(&[descriptor.clone()]) else {
+        let desc = std::slice::from_ref(&descriptor);
```

Comment:
I feel very dirty adding a comment on a two lines PR, but could you drop the `std::` prefix? It's part of the code conventions for this codebase